stage.css

```diff
@@ -1,12 +1,12 @@
 html, body {
   margin: 0;
-  min-height: 100%;
-  padding: 0;
+  height: 100%;
   overflow: hidden;
+  positon: fixed;
```

Comment:
Could you sort the lines on that one? Just `height` above `margin` I think. Also fix your typo, it should be `position`, not `positon`.